### ISSUE TITLE
Compact buttons

### ISSUE THREAD

**Issue by StioStudio** [ScratchAddons/ScratchAddons#5839]
### Why this would be helpful

Not so much text

### How this addon works

Only show the icon instead like "(icon)see innside"

### Possible alternatives

I don't think so

### Additional context

I have allready made it :D
(Or a beta version of it)

**Comment by WorldLanguages** [comment]
Most discussion is happening on #5840
This issue forgot to mention that some buttons (such as "share") don't have icons on them, so I think the issue author is suggesting to include our own opinionated icons instead.

**Comment by StioStudio** [comment]
> Most discussion is happening on #5840 This issue forgot to mention that some buttons (such as "share") don't have icons on them, so I think the issue author is suggesting to include our own opinionated icons instead.

Yes, but on the pr, I allready have made the icons (they can of course be changed though)

**Comment by Samq64** [comment]
I think this could stay open.
